### PR TITLE
chore: parse role tags

### PR DESCRIPTION
**Pull request by kian99**
## Description

While working on rebac-admin for roles I ran into an error because the role tag could not be parsed. This introduces support for parsing role tags.

## Engineering checklist
*Check only items that apply*

- [ ] Documentation updated
- [x] Covered by unit tests
- [ ] Covered by integration tests

### CODE REVIEW

**Review by ale8k — APPROVED** [review]
ah bloody hell sorry! good work!

File: internal/jimm/access.go [hunk 1]
```diff
@@ -450,6 +450,15 @@ func (j *JIMM) ToJAASTag(ctx context.Context, tag *ofganames.Tag, resolveUUIDs b
...
 		return tagToString(jimmnames.GroupTagKind, group.Name), nil
+	case jimmnames.RoleTagKind:
+		role := dbmodel.RoleEntry{
+			UUID: tag.ID,
+		}
+		err := j.Database.GetRole(ctx, &role)
+		if err != nil {
+			return "", errors.E(err, fmt.Sprintf("failed to fetch role information: %s", role.UUID))
+		}
+		return tagToString(jimmnames.GroupTagKind, role.Name), nil
```
Comment (SimoneDutto) [inline internal/jimm/access.go]: here should be `RoleTagKind`

File: internal/openfga/names/names.go [hunk 2]
```diff
@@ -171,6 +171,8 @@ func ParseRelation(relationString string) (cofga.Relation, error) {
 		return CanAddModelRelation, nil
 	case AuditLogViewerRelation.String():
 		return AuditLogViewerRelation, nil
+	case AssigneeRelation.String():
```
Comment (SimoneDutto) [inline internal/openfga/names/names.go:174]: would you mind adding a test for this as well?

**Review by SimoneDutto — CHANGES_REQUESTED** [review]

File: internal/openfga/names/names.go (hunk 2 above)
Comment (kian99) [reply internal/openfga/names/names.go:174]: Added

File: internal/jimm/access.go (hunk 1 above)
Comment (kian99) [reply internal/jimm/access.go]: Good spot, fixed. I have a test for this in a follow up.

**Review by SimoneDutto — APPROVED** [review]
ty

**Comment by kian99** [comment]
Done by #1467

**Pull request closed without merging**